src/InsetOrderOptimizer.cpp

```diff
@@ -216,6 +218,25 @@ BinJunctions InsetOrderOptimizer::variableWidthPathToBinJunctions(const Variable
             }
         }
     }
+
+    if (alternate_inset_direction) {
```

Comment:
code-style bracket on newline
```suggestion
    if (alternate_inset_direction)
    {
```